test/SIL.XForge.Identity.Tests/Controllers/Account/AccountControllerTests.cs

```diff
@@ -97,6 +97,43 @@ public async Task ForgotPassword_IncorrectUsernameOrEmail()
             Assert.IsTrue(model.EnableErrorMessage == true);
         }
 
+        [Test]
+        public async Task ForgotPassword_UpdateResetPasswordKeyAndExpirationDate()
```

Comment:
I would combine this test with the `ForgotPassword_CorrectUsernameOrEmail` test. I don't think we need to separate tests.

Reply:
We added this test with CorrectusernameOrEmail test.